src/main/java/com/nike/vault/client/VaultClient.java

```diff
@@ -313,6 +333,8 @@ protected Response execute(final HttpUrl url, final String method, final Object
      * @param response Response to parses the error details from
      */
     protected void parseAndThrowErrorResponse(final Response response) {
+        logger.debug(LOG_HTTP_RESPONSE_BODY_TMPL, gson.toJson(response.body()));
```

Comment:
Yes, we saw errors but not with this. The errors we saw occurred because the response body could not be deserialized into an ErrorResponse object.

The call here is serializing a ResponseBody object into a JSON string